lib/Models/Catalog/Ows/WebFeatureServiceCapabilities.ts

```diff
@@ -162,10 +163,12 @@ function getSrsNames(json: any): SrsNamesForLayer[] | undefined {
  */
 function buildSrsNameObject(layer: any): SrsNamesForLayer {
   let srsNames: string[] = [];
-  srsNames.push(layer.DefaultSRS);
-  layer.OtherSRS?.forEach((item: string) => {
-    srsNames.push(item);
-  });
+
+  if (isJsonString(layer.DefaultSRS)) srsNames.push(layer.DefaultSRS);
+  if (Array.isArray(layer.OtherSRS))
+    layer.OtherSRS.forEach((item: string) => {
+      if (isJsonString(item)) srsNames.push(item);
+    });
```

Comment:
If layer.OtherSRS is defined but not an array, would it not be 1 string that should be added to srsNames? It would also be nice to have unit tests for this function.

Reply:
Good pickup! I have fixed it

Reply:
I hath added unit tests